CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 - [BREAKING] Consolidate default account filepath for bundled bootstrap and faucet start commands to `account.mac` (#976).
 - [BREAKING] Remove default value account filepath for faucet commands and rename --output-path to --output (#976).
 - Exposed `miden-proving-service` as a library (#956).
+- [BREAKING] Added PoW to faucet API Keys (#974).
```

Comment:
This sounds a bit odd, we aren't adding PoW to the API Keys itself. I would phrase like:
```suggestion
- [BREAKING] Enforce `PoW` on all faucet API key-authenticated requests (#974).
```